Podfile

```diff
@@ -10,6 +10,14 @@ use_frameworks!
 install! 'cocoapods',
          :deterministic_uuids => false
 
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    target.build_configurations.each do |config|
+      config.build_settings.delete 'IPHONEOS_DEPLOYMENT_TARGET'
```

Comment:
I know this fix is WIP. But I do remember using this in my podfile before, and some pods do not like the Deployment Target being removed. This can cause quite a fuss and errors among the pods unless there is potentially other fixes to follow this which may fix the fact that some pods complain about a removed Deployment Target.

A lot of pods require a Deployment target to be defined.